### ISSUE TITLE
cuDNN error

### ISSUE THREAD

**Issue by liuyanice** [Qsingle/LearnablePromptSAM#12]
![image](https://github.com/Qsingle/LearnablePromptSAM/assets/21191835/1f9f1e9b-527c-48d9-9472-2202f0624708)
which version of pytorch is used, could you share the package lists?

**Comment by Qsingle** [comment]
Good issue, we use torch 1.13.1+CU117. It seems that the model or data can not be feed into the GPU. I suggest you check the version of CUDA.

**Comment by liuyanice** [comment]
my issue：  torch 1.13.1+CU117   nvidia-smi:  CUDA Version:11.7   Driver Version:515.105.01                nvcc: cuda_11.3.r11.3

cuDNN error: CUDNN_STATUS_NOT_INITIALIZED

**Comment by Qsingle** [comment]
Can you tell me the GPU you use? The version of cudnn. If you use Anaconda or Minianaconda I suggest you to create one virtual environment by the following command. Or you can install the CUDA 11.7 and Cudnn 8.5.0.
```shell
conda create -n pytorch python=3.8 pytorch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 pytorch-cuda=11.7 -c pytorch -c nvidia
conda activate pytorch #activate the enviroment
```

**Comment by liuyanice** [comment]
RTX TI3090  single GPU,  can you tell me the GPU you use?

**Comment by liuyanice** [comment]
![image](https://github.com/Qsingle/LearnablePromptSAM/assets/21191835/b22211fe-fa8a-47a9-aa91-7f22f630fd0c)

**Comment by liuyanice** [comment]
![image2](https://github.com/Qsingle/LearnablePromptSAM/assets/21191835/a127f526-4770-40e4-9877-0b1fb65644bc)

**Comment by liuyanice** [comment]
![image3](https://github.com/Qsingle/LearnablePromptSAM/assets/21191835/ccdc7d3c-caed-44ee-b103-fccf68c6c642)

**Comment by Qsingle** [comment]
I know the reason for this error. Similar to #6, your label may contain an invalid value like 255.

**Comment by liuyanice** [comment]
Can you share your datasets, I want to reproduce the results.

**Comment by liuyanice** [comment]
Beacause I have set num classes is 2  and  used  the  blood vessel datasets.  It also displays "cudnn error".

**Comment by liuyanice** [comment]
Thank you,  I have already solved this problem.

**Comment by liuyanice** [comment]
It is indeed caused by an invalid value

**Comment by liuyanice** [comment]
I only used five images and corresponding labels for training, and the results were not good.

**Comment by liuyanice** [comment]
If I want to achieve good segmentation results, do I still need to rely on a large amount of training image data for training？
